### ISSUE TITLE
Strange behavior while debugging with breakpoints in function calls (ES6/babel)

### ISSUE THREAD

**Issue by otbe** [node-inspector/node-inspector#672]
Hi,

im using node-inspector for some ES6/7 (babel) projects, but there are some problems with breakpoints in function calls. First: Im using babel 5.5.6, node 0.12.2 and node-inspector 0.10.1. It seems that breakpoints within functions will not be triggered until I go into this function (F11) manually. Let me show you a small example:

File src/a.js
```
import { test } from './b';

let i = 5;

function test2(i) {
    let s = i;
    s += 5;
    return s;
}

let s = test(i);
let s2 = test2(i);
```

File src/b.js
```
export function test(i) {
    let s = i;
    s += 5;
    return s;
}
```

Now I compile this with ```$ babel --source-maps --out-dir dist src``` and run it with ```$ node --debug-brk dist/a.js ```. If I set breakpoints in test2() in src/a.js and test() in src/b.js only the breakpoint in a.js will be triggered. I tried to use inlined/external source maps or run it with babel-node, but the result is always the same. I can only get this working if i use F11 to go manually into test(). After doing this the breakpoints in test() (b.js) will work as expected.

If I "transpile" this short code by myself to ES5 it works as expected.

Whats wrong with my approach?

Thanks.

**Comment by 3y3** [comment]
Can you confirm this issue with ni 0.10.2?

**Comment by 3y3** [comment]
Ok, I see that you use node `0.12.x` there is a [known issue](https://github.com/joyent/node/issues/25266) with `afterCompile` event.
So, we can't handle new scripts in app process -> we can't add them to resource tree.
One only way to receive new sources - step in.

As a workaround you can use iojs or `debugger` statement.

**Comment by 3y3** [comment]
Will be fixed in 0.11.1
